backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/edges/AbstractDefinitionOwnedUsageEdgeDescriptionProvider.java

```diff
@@ -127,7 +128,18 @@ private EdgeStyle createEdgeStyle() {
     @Override
     protected boolean isDeletable() {
         // composition edges are not deletable
```

Comment:
Please remove this comment as it is not true anymore

Reply:
done